<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распаковка и копирование</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="lblDest">
       <property name="text">
        <string>Куда распаковать</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listSource"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblSource">
       <property name="text">
        <string>Папка с архивами</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listDest">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSettings">
       <property name="text">
        <string>Настройка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopy">
       <property name="text">
        <string>Копировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
